src/service/viewer-impl.js

```diff
@@ -137,7 +137,7 @@ export class Viewer {
     /** @private {!Observable} */
     this.visibilityObservable_ = new Observable();
 
-    /** @private {!Observable} */
+    /** @private {!Observable<!{paddingTop:number, duration:number|undefined, curve:string|undefined}>} */
```

Comment:
`gulp check-types` complains here as well. I think you may need to put `number|undefined` in parens. E.g. `duration:(number|undefined)`